dashboard/scripts/update_under_13_students.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+# Update all students under age 13, or with age not set, to have sharing_disabled = true.
+# sharing_disabled is contained in properties of the User model
+require_relative('../config/environment')
+
+CDO.log.info "Starting to batch update all students under 13"
+number_under_13 = 0
+
+User.find_in_batches do |users_batch|
+  ActiveRecord::Base.transaction do
+    users_batch.each do |user|
+      next unless user.under_13?
+      user.sharing_disabled = true
+      save_result = user.save
+      number_under_13 += 1
+      raise "ERROR: ID #{user.id}." unless save_result
```

Comment:
is this the best way to handle error?

Reply:
I took the structure from this one: https://github.com/code-dot-org/code-dot-org/blob/005d9f6bf93a0028cfbf6c0eb6994c988f241364/dashboard/db/migrate/20170923000355_populate_storage_id.rb#L10

Reply:
Why not just call `update!` and let that raise an error if applicable:

starting on line 13
```ruby
user.update! sharing_disabled: true
number_under_13 += 1
```